src/tools/tidy/src/style.rs

```diff
@@ -79,11 +79,11 @@ fn line_is_url(line: &str) -> bool {
                 => state = EXP_URL,
 
             (EXP_LINK_LABEL_OR_URL, w)
-                if w.starts_with("http://") || w.starts_with("https://")
+                if w.starts_with("http://") || w.starts_with("https://") || w.starts_with("../")
```

Comment:
Hmm, so this change will permit long lines anywhere that there is a word beginning with `../`. I'm not *sure* if this is what we want. Maybe it is, or maybe it's too accepting.

On the other hand, if we only keep the *second* change, below, then the `[deref-coercions]:` link in the relevant PR would put us into the `EXP_URL` state, and *there* we can accept `../`, where the context is a bit less ambiguous. 

I could go either way, just saying that maybe we don't want this line.